peyvandak/Majlesi/django-docker-compose-url-shortener/urlshortener_api/api/models.py

```diff
@@ -0,0 +1,54 @@
+from ipaddress import ip_address
+from pyexpat import model
+from statistics import mode
+from django.db import models
+from .utils import create_shortened_url
+from django.contrib.auth.models import User
+
+
+class VisitUrl(models.Model):
+    short_url = models.CharField(max_length=15, blank=True)
+    visit_date = models.DateTimeField(auto_now_add=True)
+    response_date = models.DateTimeField(auto_now_add=True)
+    visitor = models.CharField(max_length=20)
+
+    device = models.CharField(max_length=15)
+    browser = models.CharField(max_length=15)
+    browser_version = models.CharField(max_length=15)
+    os = models.CharField(max_length=15)
+    os_version = models.CharField(max_length=10)
+    ip_address = models. GenericIPAddressField(blank=True, null=True)
+
+    class Meta:
+        ordering = ["-visit_date"]
+
+
+class Shortener(models.Model):
+    created_date = models.DateTimeField(auto_now_add=True)
+
```

Comment:
Why is there a blank line between the fields of this model?
instead of:
created_date = models.DateTimeField(auto_now_add=True)

times_followed = models.PositiveIntegerField(default=0)

You could simply write:
created_date = models.DateTimeField(auto_now_add=True)
times_followed = models.PositiveIntegerField(default=0)

that blank line seems redundant